circe/decoupled_pricing/compute.py

```diff
@@ -467,39 +488,49 @@ def process_IN_CLOSE_WRITE(self, event):
         key = (home_id,task_name,input_name)
         flag = next_mul[key][0]
         if next_tasks_map[task_name][0] in home_ids: 
-            print('----- next step is home')
+            logging.debug('----- next step is home')
             #send runtime info
             
             runtime_info = 'rt_finish '+ input_name + ' '+str(ts)
             send_runtime_profile_computingnode(runtime_info,task_name,home_id)
             transfer_data(home_id,username,password,event.pathname, "/output/"+new_file)   
         else:
-            print('----- next step is not home')
+            logging.debug('----- next step is not home')
             while len(global_task_node_map)==0:
-                print('Global task mapping is not loaded')
+                logging.debug('Global task mapping is not loaded')
                 time.sleep(1)
-            print('Current mapping input list')
+            logging.debug('Current mapping input list')
             next_hosts = [global_task_node_map[mapping_input_id[(home_id,input_name)],home_id,x] for x in next_tasks_map[task_name]]
             
-            print('Sending the output files to the corresponding destinations')
-            print(next_hosts)
+            logging.debug('Sending the output files to the corresponding destinations')
+            logging.debug(next_hosts)
             if flag=='true': 
-                print('not wait, send')
+                logging.debug('not wait, send')
+                
                 runtime_info = 'rt_finish '+ input_name + ' '+str(ts)
                 send_runtime_profile_computingnode(runtime_info,task_name,home_id)
 
                 #send a single output of the task to all its children 
+                # Using unicast 
+                # destinations = ["/centralized_scheduler/input/" +x + "/"+home_id+"/"+new_file for x in next_tasks_map[task_name]]
```

Comment:
Delete commented code. I'm assuming this is in the non-multicast version somewhere else in the code, so we don't need the commented code here.

Reply:
I think it is still better to keep the unicast option temporarily. We may need to introduce unicast/ multicast option in the configuration file in the future if neccessary.